Greenshot/Drawing/Adorners/MoveAdorner.cs

```diff
@@ -147,12 +147,13 @@ public override void Paint(PaintEventArgs paintEventArgs)
 
 			targetGraphics.SmoothingMode = SmoothingMode.None;
 			targetGraphics.CompositingMode = CompositingMode.SourceCopy;
-			targetGraphics.PixelOffsetMode = PixelOffsetMode.Half;
+			targetGraphics.PixelOffsetMode = PixelOffsetMode.None;

```

Comment:
@Lakritzator is there a special reason this needed/needs to be set to `Half``? :thinking:  it caused a weird drawing bugs when I added the 1px border (repaint artefacts + a single missing pixel). Got those fixed by changing to `None`, I don't think it broke anything else.

Reply:
I'm pretty sure this was the standard, as Half normally improves the quality.
See about it here: https://photosauce.net/blog/post/image-scaling-with-gdi-part-3-drawimage-and-the-settings-that-affect-it

Reply:
> None=Bad, Half=Good

Hmm, maybe I am missing something, but I am not sure whether this article really applies to our scenario here. The blog post is all about scaling images, and so are most other examples I found dealing with `PixelOffsetMode.Half`. Actually, as we are drawing geomatrical shapes using integer coordinates, I'd rather be surprised if we really had to think about pixel offset, smoothing and interpolation at all... :thinking: 

I noticed that `TargetAdorner` had none of them set, so I tried to discard all three from `ResizeAdorner` and `MoveAdorner` too. The result looks as I'd expect it to look. So maybe we don't need them in this scenario at all?